feat(content): Quicksilver package mixup (#8350)
endless-sky/endless-sky: data/human/human missions.txt

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

mission "Quicksilver Mixup [0]"
	name "Package to Silver"

> 	description "Deliver a package weighing <tons> to Silver by <date>. Payment is <payment>."

Author: Hahaha, I was thinking at home what possibly he might be doing to carry a one ton package. But you do go and handle those Deep cubes with your hands and they're a ton.

			branch "can't make it in time"
				not "hyperjumps to system: 3"
			`	It's going to be close to his deadline, but it's possible.`
				goto choice
			label "can't make it in time"
			`	How he expects you to deliver that package in three days is beyond you, seeing as Silver is not within three jumps of <origin>, but you might as well try and get there as soon as possible.`
			label choice
			choice
				`	(Deliver the package.)`
				`	(Find another pilot to give the package to.)`
					goto declined
			`	You take the package to your ship, leave it in the cargo bay, and prepare to fly to Silver.`
				accept
			
			label declined
			`	It takes a while searching for a pilot who can handle the delivery, but you do eventually find one. You hand them the package, explain the situation, thank them, and leave.`
				decline
	on visit
		dialog phrase "generic cargo on visit"

mission "Quicksilver Mixup [1]"
	name "Redirect Package to Quicksilver"
	description "Apparently the package of <tons> labeled for Silver was actually intended for Quicksilver. You'll have missed the deadline, so you probably won't be paid the <payment> offered."
	landing
	cargo package 1

> 	"apparent payment" 12100

Reviewer: ```suggestion
	"apparent payment" 17100
```
It was 17100 before, and with the bonus 5k it is 22100, so I guess this one is wrong.

	source Silver
	destination Quicksilver
	to offer
		or
			has "Quicksilver Mixup [0]: done"
			has "Quicksilver Mixup [0]: failed"
	on offer
		conversation
			`You take the package to the post office in Silver's spaceport, and the service clerk immediately lets out a sigh of frustration before setting the package down on the counter in front of you.`
			`	"I'm sorry, but we can't take this package," they say.`
			choice
				`	"Is something wrong?"`
					goto misdirected
				`	(Insist that they send it.)`

> 			`	"Again, I'm sorry, but we can't send the package," they reiterate. "It's got an incorrect address.`

Reviewer: ```suggestion
			`	"Again, I'm sorry, but we can't send the package," they reiterate. "It has an incorrect address.`
```
The tone feels too informal for a post office.

Author: I don't know, he continues to be informal later on. It might just be the way people from Silver talk.

			label misdirected

> 			`	"This package is addressed to 2081 Amethyst Avenue, Mercury City, Silver. Mercury City is Quicksilver's capital. Y'know, that planet out east in Algol. Syndicate space. Not a place on Silver. The package was posted to the wrong planet."`

Reviewer: ```suggestion
			`	"This package is addressed to 2081 Amethyst Avenue, Mercury City, <origin>. Mercury City is <planet>'s capital. Y'know, that planet out east in Algol. Syndicate space. Not a place on <origin>. The package was posted to the wrong planet."`
```

			`	"Oh," you say.`

> 			`	"I've lost count of how many times this has happened just this month. Silver, Quicksilver, I wonder what's the difference! At this point, I don't know why no one's started to regularly meet with Quicksilver's postmaster to exchange all this wrongly-posted mail. Alright, here's a word of advice: I highly recommend you take a picture of the incorrect address. I've had plenty of captains get put on blast 'cause of an error the sender made. Not your fault it got addressed to the wrong planet."`

Reviewer: ```suggestion
			`	"I've lost count of how many times this has happened just this month. <origin>, <planet>, I wonder what's the difference! At this point, I don't know why no one's started to regularly meet with <planet>'s postmaster to exchange all this wrongly-posted mail. Alright, here's a word of advice: I highly recommend you take a picture of the incorrect address. I've had plenty of captains get put on blast 'cause of an error the sender made. Not your fault it got addressed to the wrong planet."`
```

			`	You thank the clerk and take the package back to your ship.`
			choice
				`	(Take a picture of the incorrect address.)`
				`	(Don't bother taking a picture.)`
					accept
			action
				set "took picture of quicksilver package"
			`	You take a few photographs of the address label, just to be sure that you've got a clear picture of the wrong address.`
				accept
	on complete
		conversation

> 			`When you talk to a post officer in Quicksilver's spaceport and point out the incorrect address before handing it over, you note that he exhibits the same frustration the clerk on Silver did regarding the frequent switch-ups. He takes the package, muttering something about offworlders "not knowing the difference even if it smacked 'em in the face" as he did.`

Reviewer: ```suggestion
			`When you talk to a post officer in <planet>'s spaceport and point out the incorrect address before handing it over, you note that he exhibits the same frustration the clerk on <origin> did regarding the frequent switch-ups. He takes the package, muttering something about off-worlders "not knowing the difference even if it smacked 'em in the face" as he did.`
```
Again, it feels too informal.

			`	Once you return to your ship, you notice that the man who gave you that package had sent you a message. Or, to be more precise, a series of messages. An angry series of messages.`
			`	"WHAT TOOK YOU SO LONG???" one of them said. "I NEEDED THAT DELIVERED DAYS AGO", says the next one.`
			branch picture "didn't take picture"
				has "took picture of quicksilver package"

			label picture

> 			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he place the correct address.`

Reviewer: ```suggestion
			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he placed the correct address.`
```

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

mission "Quicksilver Mixup [0]"
	name "Package to Silver"
	description "Deliver a package weighing <tons> to Silver by <date>. Payment is <payment>."
	"apparent payment" 17100
	cargo package 1
	deadline 3 0
	to offer
		random < 25
	source
		near Algol 3
	destination Silver
	on offer
		conversation
			`You're looking at some job listings when a man with a harried face approaches you, carrying a package.`
			`	"Excuse me, Captain," he says. "I need this package delivered in the next three days. Details are on the package. I'll pay you <payment> once it's done." Without waiting for a response, he shoves the package into your hands and runs off to attend to whatever business has gotten him in such a hurry. The package is addressed to a location on Silver.`
			branch "can't make it in time"
				not "hyperjumps to system: 3"

> 			`	It's going to be close to his deadline, but it's possible.`

Reviewer 2: ```suggestion
			`	It's going to be close to his deadline, but it's possible.`
				goto choice
```

			label "can't make it in time"

> 			`	How he expects you to deliver that package in three days is beyond you, seeing as Silver is not within three jumps of <origin>, but you might as well try and get there as soon as possible.`
> 			choice

Reviewer 2: ```suggestion
			`	How he expects you to deliver that package in three days is beyond you, seeing as Silver is not within three jumps of <origin>, but you might as well try and get there as soon as possible.`
			label choice
			choice
```

Reviewer 2: Need to jump over the mention of an impossible deadline if you do meet it.

Author: Is it necessary? Even if it is possible, the point is that it went the wrong way.

Reviewer 2: You don't know that at that point in the conversation, though. If you can make the deadline, then <origin> is within 3 jumps of Silver and it's possible. You want to skip that block of text if you get the mission in a location like that.

Author: Isn't that covered by 
```
		branch "can't make it in time"
				not "hyperjumps to system: 3"
			`	It's going to be close to his deadline, but it's possible.`
```
I could also add a condition for the player having a jump drive too.

Reviewer 2: Conversations continuing going by line line if nothing interrupts them. If you're within the jumps needed, it'll say "It's going to be close, etc.", _then_ continue on to the next block of text, saying it's impossible.

But you've already made the suggestion change here of adding a `label choice`. So it's good.

Author: Oh, I see. : )

				`	(Deliver the package.)`
				`	(Find another pilot to give the package to.)`
					goto declined
			`	You take the package to your ship, leave it in the cargo bay, and prepare to fly to Silver.`
				accept
			
			label declined
			`	It takes a while searching for a pilot who can handle the delivery, but you do eventually find one. You hand them the package, explain the situation, thank them, and leave.`
				decline
	on visit
		dialog phrase "generic cargo on visit"

mission "Quicksilver Mixup [1]"
	name "Redirect Package to Quicksilver"
	description "Apparently the package of <tons> labeled for Silver was actually intended for Quicksilver. You'll have missed the deadline, so you probably won't be paid the <payment> offered."
	landing
	cargo package 1
	"apparent payment" 12100
	source Silver
	destination Quicksilver
	to offer

> 		has "Quicksilver Mixup [0]: done"
> 		has "Quicksilver Mixup [0]: failed"

Reviewer 2: ```suggestion
		or
			has "Quicksilver Mixup [0]: done"
			has "Quicksilver Mixup [0]: failed"
```

			`	You thank the clerk and take the package back to your ship.`
			choice
				`	(Take a picture of the incorrect address.)`
				`	(Don't bother taking a picture.)`
					accept
			action
				set "took picture of quicksilver package"
			`	You take a few photographs of the address label, just to be sure that you've got a clear picture of the wrong address.`
				accept
	on complete
		conversation
			`When you talk to a post officer in Quicksilver's spaceport and point out the incorrect address before handing it over, you note that he exhibits the same frustration the clerk on Silver did regarding the frequent switch-ups. He takes the package, muttering something about offworlders "not knowing the difference even if it smacked 'em in the face" as he did.`
			`	Once you return to your ship, you notice that the man who gave you that package had sent you a message. Or, to be more precise, a series of messages. An angry series of messages.`
			`	"WHAT TOOK YOU SO LONG???" one of them said. "I NEEDED THAT DELIVERED DAYS AGO", says the next one.`
			branch picture "didn't take picture"
				has "took picture of quicksilver package"

> 			label picture
> 			branch "picture success" "picture failure"
> 				random < 50
> 
> 			label "picture failure"
> 			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he place the correct address.`
> 			`	He leaves you on read.`
> 
> 			label "picture success"
> 			action
> 				payment 22100
> 			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he place the correct address.`
> 			`	He sees the message, but doesn't send a reply. Just before you walk away from your ship's terminal, however, you see a new message appear:`
> 			`	"I am so sorry."`
> 			`	The man then proceeds to apologize profusely for his mistake and his outburst, offering to pay an extra 5000 credits for the trouble. You gladly accept.`
> 
> 			label "didn't take picture"
> 			`	You try to defend yourself, but the man remains adamant that it was your mistake that got the package sent so late. After a frustrating back-and-forth, you decide to relent and (reluctantly) offer an apology. He ignores it and threatens to get spaceport authorities to "STOP OFFERING JOBS TO INCOMPETENTS LIKE YOU."`
> 			`	You don't bother to reply.`

Reviewer 2: ```suggestion
			label picture
			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he place the correct address.`
			branch "picture success" "picture failure"
				random < 50

			label "picture failure"
			`	He leaves you on read.`
				goto end

			label "picture success"
			action
				payment 22100
			`	He sees the message, but doesn't send a reply. Just before you walk away from your ship's terminal, however, you see a new message appear:`
			`	"I am so sorry."`
			`	The man then proceeds to apologize profusely for his mistake and his outburst, offering to pay an extra 5000 credits for the trouble. You gladly accept.`
				goto end

			label "didn't take picture"
			`	You try to defend yourself, but the man remains adamant that it was your mistake that got the package sent so late. After a frustrating back-and-forth, you decide to relent and (reluctantly) offer an apology. He ignores it and threatens to get spaceport authorities to "STOP OFFERING JOBS TO INCOMPETENTS LIKE YOU."`
			`	You don't bother to reply.`
			
			label end
```

Reviewer 2: Can share the picture comment before it branches, then need an end label to jump to after each.

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

mission "Quicksilver Mixup 0"
	minor
	name "Package to Silver"
	description "Deliver a package to <planet> by <date>. Payment is <payment>."
	"apparent payment" 17100
	cargo package 1
	deadline 4 0
	to offer
		random < 25
	source

> 		near Algol 3

Reviewer 3: ```suggestion
		near Algol 1 3
```

			`When you talk to a post officer in <planet>'s spaceport and point out the incorrect address before handing it over, you note that he exhibits the same frustration the clerk on <origin> did regarding the frequent switch-ups. He takes the package, muttering something about off-worlders "not knowing the difference even if it smacked 'em in the face" as he did.`
			`	Once you return to your ship, you notice that the man who gave you that package had sent you a message. Or, to be more precise, a series of messages. An angry series of messages.`
			`	"WHAT TOOK YOU SO LONG???" one of them said. "I NEEDED THAT DELIVERED DAYS AGO", says the next one.`
			branch picture "didn't take picture"
				has "took picture of quicksilver package"

			label picture
			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he placed the correct address.`
			branch "picture success" "picture failure"
				random < 50

			label "picture failure"
			action
				payment -17100

> 			`	He leaves you on read.`

Reviewer 4: "on read"? Was this supposed to be "unread"?

```suggestion
			`	He leaves the message unread.`
```

Author: It's a joke (although not one I would have made). You can leave a message unread and apps like WhatsApp will say that you've seen and read the message, but you haven't replied, which is meant to be disrespectful or something.

Reviewer 4: Then let's make it clearer in another way:

```suggestion
			`	He doesn't reply, though you can see that he read the message..`
```

Author: The joke explicitly is leaving someone "on read", so this suggestion does take the joke out of it. I don't mind the change, but I'll talk with Improvise who wrote the story whether they would like this suggestion or not.

Reviewer 4: Maybe my age is showing, but "leaving someone on read" is slang I've never heard before. I understand what it means now, but for someone of my vintage, it read like a typo. :)

Reviewer 5: That suggestion very much does remove the joke. 
So I'd suggest either leaving it as it is, and keeping the humor in, or rewriting it entirely to not be a joke at all. 

It looks like "leave on read" has been slang since ~2019, so I would personally not have any concerns with leaving it as it is. It's humorous.

Reviewer 4: What's the joke? That he reads your message doesn't reply to it? Can someone explain what the joke is?

Reviewer 5: The joke here is that the client in question was upset at you, and was sending you a string of angry messages over something that was ultimately their own fault. 
If you point out to them that it was their own fault, and not yours, they suddenly go silent and "leave you on read". 

The sudden shift in dynamic between you and your client is what is supposed to be funny, and that sudden shift relies on the relatively short punchline. 

If you reword the punchline to something that still technically means the same thing, but says something different, it doesn't have the impact required to function as a punchline and the build-up and payoff of the joke both fall flat. 

So if we want to keep the joke, we should keep it as it is, if the slang isn't acceptable, then (in my opinion) it should just be rewritten to not include the joke at all, as no joke is better than a joke that falls flat.

Reviewer 4: I hate to be that guy, but what you described isn't a joke. It may be intended to be _funny_, but it's not a _joke_. All jokes are (or should be) funny, but not everything that's funny is a joke.

And I didn't find it funny due to the use of unfamiliar slang causing me to interpret the final text as a typo. In general this game doesn't use a lot of modern slang in descriptions, and I think it would be best to continue that trend here to preserve clarity. If that requires re-doing some of the text to preserve the humor, so be it.

I've said my peace on this topic and made my opinion known; I'll let others decide how to proceed now.

Reviewer 6: It's certainly not what i'd call a joke, but it's a very common thing to say, and to leave FL on read fits perfectly with what the space karen would do, so i see no reason to change that whole conversation structure. 

I don't think many, if anybody, would have issue understanding, due to how widespread that term is. It is currently a 7 year old expression (originated in 2016 according to dictionary.com) with a lot of staying power. It isn't slang as much a a common everyday expression - newer and less used words have been included in dictionaries. Hell, there's references to furries and fursuits in Endless Sky, which, while have existed before, have become commonly known as what they are referenced as in game more recently, as well, if i am not mistaken. I mean, my autocorrect is tagging both "furries" and "fursuits" as words that don't exist, and i find it much more likely for a player not to know of them than to not know of this expression. I don't see an issue with using "on read" at all.

If we must change it, then I'd write it as `He doesn't respond.`. It would keep the fitting structure. But these are my two cents on how we should proceed and why i think that!

Author: Well, I can't tell you what slang will be like in 1000 years' time, and Improvise hasn't got back to me yet. I'll change it to `He doesn't respond`.

			`	You thank the clerk and take the package back to your ship.`
			choice
				`	(Take a picture of the incorrect address.)`
				`	(Don't bother taking a picture.)`
					accept
			action
				set "took picture of quicksilver package"
			`	You take a few photographs of the address label, just to be sure that you've got a clear picture of the wrong address.`
				accept
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		payment 17100
		event "quicksilver mixup 2 available" 90
		conversation
			`When you talk to a post officer in <planet>'s spaceport and point out the incorrect address before handing it over, you note that he exhibits the same frustration the clerk on <origin> did regarding the frequent switch-ups. He takes the package, muttering something about off-worlders "not knowing the difference even if it smacked 'em in the face" as he did.`
			`	Once you return to your ship, you notice that the man who gave you that package had sent you a message. Or, to be more precise, a series of messages. An angry series of messages.`

> 			`	"WHAT TOOK YOU SO LONG???" one of them says. "I NEEDED THAT DELIVERED DAYS AGO", says the next one.`

Reviewer 3: I think putting each message on a new line looks better.
```suggestion
			`	"WHAT TOOK YOU SO LONG???"`
			`	"I NEEDED THAT DELIVERED DAYS AGO"`
```

			branch picture "didn't take picture"
				has "took picture of quicksilver package"

			label picture
			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he placed the correct address.`
			branch "picture success" "picture failure"
				random < 50

			label "picture failure"
			action
				payment -17100
			`	He doesn't respond.`

> 				goto end

Reviewer 3: Using `decline` (or another conversation end point) is functionally equivalent.
```suggestion
				decline
```

			label "picture success"
			action
				payment 5000

> 			`	He sees the message, but doesn't send a reply. But just before you walk away from your ship's terminal, you see a new message appear:`

Reviewer 3: Hanging conjunctions are bad form outside of dialogue.
```suggestion
			`	He sees the message, but doesn't send a reply. Just before you walk away from your ship's terminal, you see a new message appear:`
```

			`	"I am so sorry."`

> 			`	The man then proceeds to apologize profusely for his mistake and his outburst, offering to pay an extra 5000 credits for the trouble. You gladly accept.`

Reviewer 3: ```suggestion
			`	The man then proceeds to apologize profusely for his mistake and his outburst, offering to pay an extra 5,000 credits for the trouble. You gladly accept.`
```

> 				goto end

Reviewer 3: ```suggestion
				decline
```

			label "didn't take picture"
			action
				payment -17100
			`	You try to defend yourself, but the man remains adamant that it was your mistake that got the package sent so late. After a frustrating back-and-forth, you decide to relent and (reluctantly) offer an apology. He ignores it and threatens to get spaceport authorities to "STOP OFFERING JOBS TO INCOMPETENTS LIKE YOU."`
			`	You don't bother to reply.`
			

> 			label end

Reviewer 3: ```suggestion
```
